samples/MusicStore.Standalone/MusicStore.Standalone.csproj

```diff
@@ -0,0 +1,50 @@
+<Project Sdk="Microsoft.NET.Sdk.Web" ToolsVersion="15.0">
+
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'version.props'))\version.props" />
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'dependencies.props'))\dependencies.props" />
+
+  <PropertyGroup>
+    <Description>Music store application on ASP.NET Core</Description>
+    <Authors>Microsoft</Authors>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <DefineConstants>$(DefineConstants);DEMO;TESTING</DefineConstants>
+    <PreserveCompilationContext>true</PreserveCompilationContext>
+    <OutputType>Exe</OutputType>
+    <WarningsAsErrors>true</WarningsAsErrors>
+    <RuntimeIdentifiers>win7-x64;win7-x86;win10-x64;win10-x86;osx.10.10-x64;osx.10.11-x64;osx.10.12-x64;ubuntu.14.04-x64;ubuntu.15.04-x64;centos.7-x64;rhel.7.2-x64</RuntimeIdentifiers>
```

Comment:
I believe odd version numbers is something like a preview release, we can just add 16 and keep 15, doesn't hurt